azure-postgresql-flexible-server.yml

```diff
@@ -0,0 +1,192 @@
+# Copyright 2018 the Service Broker Project Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+version: 1
+name: csb-azure-postgresql-flexible-server
+id: d69dd4aa-e27e-490c-bdbf-c887563da27f
+description: Azure Database for PostgreSQL - flexible server
+display_name: Azure Database for PostgreSQL - flexible server
+image_url: file://service-images/csb.png
+documentation_url: https://learn.microsoft.com/en-gb/azure/postgresql/
+support_url: https://learn.microsoft.com/en-gb/azure/postgresql/
+tags: [azure, postgresql, postgres, preview, flexible server]
+plan_updateable: true
+provision:
+  plan_inputs: []
+  user_inputs:
+  - field_name: postgres_version
+    type: string
+    details: The version for the PostgreSQL instance.
+    default: "11"
```

Comment:
Would it be better not to have a default? `"11"` will look dated in time, but we won't be able to change it for fear that it will force version updates for folks who just used the default.